cli/command/image/tree.go

```diff
@@ -488,3 +490,17 @@ func widestFirstColumnValue(headers []imgColumn, images []topImage) int {
 	}
 	return width
 }
+
+func stdoutRedirected(dockerCLI command.Cli) bool {
```

Comment:
Nice! I wonder if there's more places we need things like this; it's good to keep it internal here for now, but possibly we can start moving it elsewhere (and add some tests for it).

Can you change this to use the `Streams` interface?

```suggestion
func stdoutRedirected(streams command.Streams) bool {
```